tests/clients/test_basic_client.py

```diff
@@ -3,12 +3,15 @@
 from typing import Dict, Optional, Union
 from unittest.mock import MagicMock
 
+import freezegun
 import torch
 from flwr.common import Scalar
 from freezegun import freeze_time
 
 from fl4health.clients.basic_client import BasicClient
 
+freezegun.configure(extend_ignore_list=["transformers"])  # type: ignore
```

Comment:
I had to add this to prevent `freeze_time` from failing. More details: 

https://github.com/huggingface/transformers/issues/24545

Reply:
Interesting 🤔